openedx/core/djangoapps/self_paced/models.py

```diff
@@ -1,21 +0,0 @@
-"""
-Configuration for self-paced courses.
-"""
-
-
-from config_models.models import ConfigurationModel
-from django.db.models import BooleanField
-from django.utils.translation import ugettext_lazy as _
-
-
-class SelfPacedConfiguration(ConfigurationModel):
-    """
-    Configuration for self-paced courses.
-
-    .. no_pii:
-    """
-
-    enable_course_home_improvements = BooleanField(
-        default=False,
-        verbose_name=_("Enable course home page improvements.")
-    )
```

Comment:
@davidjoy @Agrendalath 

The only current test failures are due to migrations not being in sync.  This is because of the removal of this class. According to the [database migration docs](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Howtodropatable), the removal of the model via migration should be in a separate release. As a result, should I be reverting the changes to this class and simply ensuring it isn't displayed in the Admin panel or used anywhere (and it can be removed, along with a migration, in a future release)?

Reply:
@pcockwell, that sounds reasonable to me.